dali/test/CMakeLists.txt

```diff
@@ -14,6 +14,6 @@
 
 if (BUILD_TEST)
   # get all the test srcs
-  file(GLOB tmp *_test.cc *.cu)
+  file(GLOB tmp *_test.cc *.h *.cu)
```

Comment:
AFAIK, it's not a problem in CMake to have header files listed, while thanks to it IDE can include them in project and work on them. If you feel it's harmful, I'll remove it.

Reply:
I think it is better to keep that consistent with the rest of cmake files.

Reply:
I've noticed that our CMakeLists.txt are usually very similar. I think we could extract the common statements to some cmake macros so that we don't repeat ourselves

Reply:
About macros: this will make adding new source directories easier for us - perhaps. But we're an open source project and as such, we should also care about being understandable for broader audience. I've worked with OpenCV, which essentially built some kind of DSL inside CMake. Adding anything to it (or, worse, trying to extract a part) is a nightmare.

Reply:
I see your point and I agree with putting readability first :+1: